src/components/views/symptom-checker.tsx

```diff
@@ -11,31 +11,29 @@ import {useTranslation} from 'react-i18next';
 import {RouteProp} from '@react-navigation/native';
 import {useFocusEffect} from '@react-navigation/native';
 
-import {useApplication, Symptoms} from '../../providers/context';
-import {useSettings} from '../../providers/settings';
-
-import {Spacing, Separator} from '../atoms/layout';
-import {Heading} from '../atoms/heading';
-import {Button} from '../atoms/button';
-import {Markdown} from '../atoms/markdown';
-import {Dropdown} from '../atoms/dropdown';
-import {SelectList} from '../atoms/select-list';
-import {CheckInConsent} from '../molecules/checkin-consent';
-import {LocationDropdown} from '../molecules/locality-dropdown';
-import {Result} from '../molecules/result-card';
-
-import {colors} from '../../constants/colors';
-import Layouts from '../../theme/layouts';
-import {text, shadows, baseStyles} from '../../theme';
+import {Button} from 'components/atoms/button';
+import {CheckInConsent} from 'components/molecules/checkin-consent';
+import {text, baseStyles} from 'theme';
+import {Dropdown} from 'components/atoms/dropdown';
+import {Heading} from 'components/atoms/heading';
+import {LocationDropdown} from 'components/molecules/locality-dropdown';
+import {Markdown} from 'components/atoms/markdown';
+import {Result} from 'components/molecules/result-card';
+import {SelectList} from 'components/atoms/select-list';
+import {Spacing, Separator} from 'components/atoms/layout';
+import {useApplication, Symptoms} from 'providers/context';
+import {useSettings} from 'providers/settings';
+import {Scrollable} from 'components/templates/scrollable';
+import {Card} from 'components/atoms/card';
 
 const width = Dimensions.get('window').width;
 const ANIMATION_DURATION = 300;
 
 const icons = {
-  '1': require('../../assets/images/symptoma/image.png'),
-  '2': require('../../assets/images/symptomc/image.png'),
-  '3': require('../../assets/images/symptomb/image.png'),
-  '4': require('../../assets/images/symptomd/image.png')
+  '1': require('assets/images/symptoma/image.png'),
```

Comment:
A bit confusing as to why this is numbered 1, 2, .. 4 while the images are named a, b, .. c and not in order.